File: LV2/2 x n 타일링.js
function solution(n) {
  let prev = 1;
  let curr = 1;

  for (let i = 2; i <= n; i++) {
    let next = (prev + curr) % 1000000007;
    prev = curr;
    curr = next;
  }

  return curr;
}

File: LV2/2 x n 타일링.java
class Solution {

  public int solution(int n) {
    int prev = 1;
    int curr = 1;

    for (int i = 2; i <= n; i++) {
      int next = (prev + curr) % 1000000007;
      prev = curr;
      curr = next;
    }

    return curr;
  }
}
